### PR TITLE
add avatar user

### PR DESCRIPTION
**Pull request by NgocVu1994**

### CODE REVIEW

**Comment by thiennv57** [comment]
OK

**Comment by vietmt** [comment]
Add folder /public/uploads to .gitignore and remove files upload from git

**Comment by vietmt** [comment]
OK

File: app/views/layouts/_admin_header.html.erb [hunk 1]
```diff
@@ -46,7 +46,7 @@
     <section class="sidebar">
       <div class="user-panel">
         <div class="pull-left image">
-          <%= image_tag("profile.png", alt: "User Image", class: "img-circle") %>
+          <%= image_tag current_user.avatar_url, alt: "User Image", class: "img-circle" %>
```
Comment (dieunb) [inline app/views/layouts/_admin_header.html.erb]: i18n `"User Image"`

File: app/views/layouts/_admin_header.html.erb [hunk 2]
```diff
@@ -18,12 +18,12 @@
...
           <li class="dropdown user user-menu">
             <a href="#" class="dropdown-toggle" data-toggle="dropdown">
-              <%= image_tag("profile.png", alt: "User Image", class: "user-image") %>
+              <%= image_tag current_user.avatar_url, alt: "User Image", class: "user-image" %>
               <span class="hidden-xs"><%= current_user.name %></span>
             </a>
             <ul class="dropdown-menu">
               <li class="user-header">
-                <%= image_tag("profile.png", alt: "User Image", class: "img-circle") %>
+                <%= image_tag current_user.avatar_url, alt: "User Image", class: "img-circle" %>
```
Comment (dieunb) [inline app/views/layouts/_admin_header.html.erb]: i18n `"User Image"`

File: app/views/layouts/_admin_header.html.erb [hunk 3]
```diff
@@ -18,12 +18,12 @@
           </li>
           <li class="dropdown user user-menu">
             <a href="#" class="dropdown-toggle" data-toggle="dropdown">
-              <%= image_tag("profile.png", alt: "User Image", class: "user-image") %>
+              <%= image_tag current_user.avatar_url, alt: "User Image", class: "user-image" %>
```
Comment (dieunb) [inline app/views/layouts/_admin_header.html.erb]: i18n `"User Image"`

File: app/views/layouts/_user_header.html.erb [hunk 4]
```diff
@@ -18,12 +18,12 @@
             </li>
             <li class="dropdown user user-menu">
               <a href="#" class="dropdown-toggle" data-toggle="dropdown">
-                <%= image_tag("profile.png", alt: "User Image", class: "user-image") %>
+                <%= image_tag current_user.avatar_url, alt: "User Image", class: "user-image" %>
```
Comment (dieunb) [inline app/views/layouts/_user_header.html.erb]: i18n `"User Image"`

File: app/views/layouts/_user_header.html.erb [hunk 5]
```diff
@@ -18,12 +18,12 @@
...
                 <span class="hidden-xs"><%= current_user.name %></span>
               </a>
               <ul class="dropdown-menu">
                 <li class="user-header">
-                  <%= image_tag("profile.png", alt: "User Image", class: "img-circle") %>
+                  <%= image_tag current_user.avatar_url, alt: "User Image", class: "img-circle" %>
```
Comment (dieunb) [inline app/views/layouts/_user_header.html.erb]: i18n `"User Image"`

File: app/views/users/_info.html.erb [hunk 6]
```diff
@@ -1,6 +1,6 @@
 <div class="col-md-6 ">
   <div class="form-group">
-    <%= image_tag "profile", class: "profile-image" %>
+    <%= image_tag @user.avatar_url, class: "img-circle" %>
```
Comment (dieunb) [inline app/views/users/_info.html.erb]: i18n `"User Image"`

File: app/helpers/application_helper.rb [hunk 7]
```diff
@@ -56,4 +56,9 @@ def select_profile_field target, target_array, builder
       [t.name, t.id ]}, builder.object.send("#{target}_id")),
       {include_blank: true}, class: "form-control"
   end
+
+  def user_avatar user, name_image
```
Comment (dieunb) [inline app/helpers/application_helper.rb]: `name_image => class_name`

**Pull request merged**